midend/local_copyprop.cpp

```diff
@@ -37,8 +37,13 @@ static const IR::Expression *lvalue_out(const IR::Expression *exp) {
 static const IR::Statement *makeSideEffectStatement(const IR::Expression *exp) {
     while (auto unary = exp->to<IR::Operation_Unary>())
         exp = unary->expr;
-    if (auto mc = exp->to<IR::MethodCallExpression>())
-        return new IR::MethodCallStatement(mc);
+    if (auto mc = exp->to<IR::MethodCallExpression>()) {
+        if (auto mem = mc->method->to<IR::Member>()) {
+            if (auto ext = mem->expr->type->to<IR::Type_Extern>()) {
+                // packet_in.lookahead has no side effects, so get rid of it too
```

Comment:
You may be right -- I added this because the bmv2 backend crashes on a bare `lookahead` call, and we have a testcase that has an unused lookahead.  Might want to fix that instead.